integration/nwo/cryptogen_config.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright IBM Corp All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+package nwo
+
+type HostnameData struct {
+	Prefix string
+	Index  int
+	Domain string
+}
+
+type SpecData struct {
+	Hostname   string
+	Domain     string
+	CommonName string
+}
+
+type NodeTemplate struct {
+	Count    int      `yaml:"Count"`
+	Start    int      `yaml:"Start"`
+	Hostname string   `yaml:"Hostname"`
+	SANS     []string `yaml:"SANS"`
+}
+
+type NodeSpec struct {
+	isAdmin            bool
+	Hostname           string   `yaml:"Hostname"`
+	CommonName         string   `yaml:"CommonName"`
+	Country            string   `yaml:"Country"`
+	Province           string   `yaml:"Province"`
+	Locality           string   `yaml:"Locality"`
+	OrganizationalUnit string   `yaml:"OrganizationalUnit"`
+	StreetAddress      string   `yaml:"StreetAddress"`
+	PostalCode         string   `yaml:"PostalCode"`
+	SANS               []string `yaml:"SANS"`
+	PublicKeyAlgorithm string   `yaml:"PublicKeyAlgorithm"`
+}
+
+type UsersSpec struct {
+	Count int `yaml:"Count"`
+}
+
+type OrgSpec struct {
+	Name          string       `yaml:"Name"`
+	Domain        string       `yaml:"Domain"`
+	EnableNodeOUs bool         `yaml:"EnableNodeOUs"`
+	CA            NodeSpec     `yaml:"CA"`
+	Template      NodeTemplate `yaml:"Template"`
+	Specs         []NodeSpec   `yaml:"Specs"`
+	Users         UsersSpec    `yaml:"Users"`
+}
+
+type CryptogenConfig struct {
```

Comment:
where is this used?

Reply:
On previous tests, I used cryptogen to give ed25519 credentials. It has no use now. It will be removed.